feat(clayui.com): Improve component docs by updating content Pt.2 (#3235)
liferay/clay: packages/clay-form/TOGGLE_SWITCH.mdx

<div class="nav-toc-absolute">
<div class="nav-toc">

-   [Composing](#composing)
    -   [Inside ClayRadioGroup](#inside-clayradiogroup)
-   [API](#api)

</div>
</div>

> ## Composing

Reviewer: Lets not use "composing" unless we are referencing our doc on it at https://clayui.com/docs/get-started/composing.html

Based on issue #3229, we can call either call this "Introduction", "Overview", or just leave it out. I think I would actually opt for just leaving no header for the first basic example that we showcase

Author: Oh, okay, I assumed the "condition" to use the `composing` heading was if we're describing using component props/subcomponents to compose a component

In order to create a controlled `Toggle` you need to provide the `toggled` and `onToggle` props.

You can also have custom icons that further signal the current state of the Toggle by providing a `symbol` prop.

<Toggle />

### Inside ClayRadioGroup

> If you want the Toggle to behave as a radio element (toggling one on will toggle others off) wrap your multiple `Toggles` with a [ClayRadioGroup](/docs/components/radio-group.html) component.

Reviewer: ```suggestion
If you want the Toggle to behave like a radio element (toggling one on will toggle others off) wrap multiple `Toggles` with a [ClayRadioGroup](/docs/components/radio-group.html) component.
```
